core/src/main/java/hudson/slaves/CloudRetentionStrategy.java

```diff
@@ -47,6 +47,20 @@ public CloudRetentionStrategy(int idleMinutes) {
         this.idleMinutes = idleMinutes;
     }
 
+    /**
+     * @since FIXME
+     */
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    /**
+     * @since FIXME
+     */
+    public void setIdleMinutes(int idleMinutes) {
```

Comment:
Then just protected? also no problem to return new object from readReasolve() using the existing constructor